### PR TITLE
Georgina fernandez/unit test

### PR DESCRIPTION
**Pull request by Georgina-Fernandez**
Unit tests for all functionalities 

App tree

appTree

Tests results

Coverage

Coverage

### CODE REVIEW

File: app/src/main/java/com/example/parkingsystem/util/ExtensionFunctions.kt [hunk 1]
```diff
@@ -5,3 +5,7 @@ import android.widget.Toast
 
 fun Context.showToast(message: String) =
     Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
+
```
Comment (SartiMau) [inline app/src/main/java/com/example/parkingsystem/util/ExtensionFunctions.kt]: remove these extra spaces pls

File: app/src/main/java/com/example/parkingsystem/mvp/view/ParkingReservationView.kt [hunk 2]
```diff
@@ -102,6 +102,7 @@ class ParkingReservationView(activity: Activity, private val binding: ActivityPa
         }
     }
 
+
```
Comment (SartiMau) [inline app/src/main/java/com/example/parkingsystem/mvp/view/ParkingReservationView.kt]: remove this extra space pls

**Review by SartiMau — APPROVED** [review]

File: app/src/main/java/com/example/parkingsystem/util/ExtensionFunctions.kt (hunk 1 above)
Comment (Georgina-Fernandez) [reply app/src/main/java/com/example/parkingsystem/util/ExtensionFunctions.kt]: Done

File: app/src/main/java/com/example/parkingsystem/mvp/view/ParkingReservationView.kt (hunk 2 above)
Comment (Georgina-Fernandez) [reply app/src/main/java/com/example/parkingsystem/mvp/view/ParkingReservationView.kt]: Done

**Review by BQuinteros — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Pull request merged**